### ISSUE TITLE
Error: Symbol pa_log_level_meta not found

### ISSUE THREAD

**Issue by hanselmandrew** [ptitSeb/box64#2220]
In my attempt to run a Valheim server with cross play enabled on an Ubuntu 22.04 ARM64 instance via Oracle Cloud, I'm hitting this error:
```
Dec 27 17:40:00 instance-20241226-2040 valheim_server.x86_64[17836]: Error: Symbol pa_log_level_meta not found, cannot apply R_X86_64_JUMP_SLOT @0x7fff06004fc0 (0x1140) in ./linux64/libpulse-mainloop-glib.so.0
Dec 27 17:40:00 instance-20241226-2040 valheim_server.x86_64[17836]: Error: relocating Plt symbols in elf libpulse-mainloop-glib.so.0
```

Trying to locate the exact library that has the hard link to the method:
```
ubuntu@instance-20241226-2040:/mnt/data/valheim/valheim_server/linux64$ nm -gCD libpulse-mainloop-glib.so.0 | grep "log_level"
                 U pa_log_level_meta
ubuntu@instance-20241226-2040:/mnt/data/valheim/valheim_server/linux64$ nm -gCD libpulse-simple.so.0 | grep "log_level"

                 U pa_log_level_meta
ubuntu@instance-20241226-2040:/mnt/data/valheim/valheim_server/linux64$ nm -gCD libpulse.so.0 | grep "log_level"

                 U pa_log_level_meta
ubuntu@instance-20241226-2040:/mnt/data/valheim/valheim_server/linux64$ nm -gCD libpulsecommon-15.99.so | grep "log_level"

00000000000385c0 T pa_log_level
0000000000030840 T pa_log_level_meta
00000000000385a0 T pa_log_levelv
000000000002fd20 T pa_log_levelv_meta
```

I see that the there are some references to the `pa_log_level_meta` method, but they are preprocessed away: 
https://github.com/ptitSeb/box64/blob/e2d7cc5011f8f88d7c96d4d395ea9927a9d9ce8e/src/wrapped/wrappedpulse.c#L48-L55 https://github.com/ptitSeb/box64/blob/e2d7cc5011f8f88d7c96d4d395ea9927a9d9ce8e/src/wrapped/wrappedpulse.c#L1470-L1482

I spent a non-zero (but small) amount of time trying to fix this myself, but I am not familiar with this code base, so I didn't make it far. 
- This block of code is 3 years old
- `myStackAlign` type signature seems to be slightly different now
- I think `pa_log_level_meta` also needs a `GO/GOM` somewhere else

**Comment by ptitSeb** [comment]
Yeah, need to wrap this function. I'll check tomorow if I can wrap it.

**Comment by hanselmandrew** [comment]
I may have spoke too soon, I realized that I was running on v2.6, so I updated to v3.2 to make sure that wasn't an issue. That was causing the server boot to repeatedly crash. Just to be sure, I installed the entire `pulselib-dev` library, and reinstalled the server via steamcmd, and I'm not seeing the reported missing symbol error anymore. So this may be a non-issue.

Still having some crashes related to some mono watchdog timer, but that seems unrelated to this library.

**Comment by LukeShortCloud** [comment]
If you're still running into crashes, please feel free to open a new GitHub Issue since the original issue has been resolved.
